### PR TITLE
[4.0] Move Users name above tabs on edit screen and remove 'Title' i.e username again.

### PR DESCRIPTION
**Pull request by tonypartridge**
Pull Request for Issue #18717 .

### Summary of Changes
Removed old title text output
Added in new fieldset
Included name above the tabs
Follows Article Edit screen layout

### Testing Instructions
Install, view  / edit users.

### CODE REVIEW

File: administrator/components/com_users/forms/user.xml [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-	<fieldset name="user_details">
-		<field 
-			name="name" 
-			type="text"
-			label="COM_USERS_USER_FIELD_NAME_LABEL"
-			required="true"
-			size="30"
+	<fieldset name="user_name">
+		<field
+				name="name"
```
Comment (Quy) [inline administrator/components/com_users/forms/user.xml]: Too much indentation.

**Comment by ghost** [comment]
I have tested this item :red_circle: unsuccessfully on e5739084b933d8755a7858b729d92518041f0a7d

#### Fieldset-Label:

bildschirmfoto 2017-11-23 um 14 06 16
#### Without PR:
bildschirmfoto 2017-11-23 um 14 08 10


<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18824">issues.joomla.org/tracker/joomla-cms/18824</a>.</sub>

**Comment by tonypartridge** [comment]
Thanks @franz-wohlkoenig 

Please re-test.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 86435a80fa715209f85a5d8b83bf68ae2b7ed0c5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18824">issues.joomla.org/tracker/joomla-cms/18824</a>.</sub>

**Comment by tonypartridge** [comment]
@brianteeman could you test pretty please?

**Comment by brianteeman** [comment]
its for j4 - i am not testing anything ui related for j4 until we have the actual ui that is going to be used.

**Comment by tonypartridge** [comment]
Yep, but it’s not really theming UI related more UX and this would pass trough on all theming.

**Comment by Quy** [comment]
For edit.php, use this instead. 
```
<?php echo JLayoutHelper::render('joomla.edit.title_alias', $this); ?>
```

**Comment by tonypartridge** [comment]
@Quy  Now updated @franz-wohlkoenig  please re-test.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 1bdeeeea3c2e207c046a8c5779ef24ec834ee020<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18824">issues.joomla.org/tracker/joomla-cms/18824</a>.</sub>

File: administrator/components/com_users/forms/user.xml [hunk 2]
```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <form>
-	<fieldset name="user_details">
-		<field 
-			name="name" 
+	<fieldset>
```
Comment (Quy) [inline administrator/components/com_users/forms/user.xml]: Do we need `<fieldset>` here considering there is only 1 field?

**Comment by tonypartridge** [comment]
Good point @Quy now updated.

**Comment by tonypartridge** [comment]
p.s. can we get re-tests.

**Comment by ggppdk** [comment]
I have tested this item :white_check_mark: successfully on 9ac523fa8cda50e6df511542a6db9a0e332213bc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18824">issues.joomla.org/tracker/joomla-cms/18824</a>.</sub>

**Comment by ggppdk** [comment]
This needs rebasing so that unit tests will run properly ?

**Comment by tonypartridge** [comment]
@ggppdk now down, can you test and also @franz-wohlkoenig  if you get a minute :-)

**Comment by tonypartridge** [comment]
Will re do at some pointing, closing for now.

**Pull request closed without merging**